server/env/.dev.env

```diff
@@ -1,3 +1,4 @@
 SECRET=SomeSecretKey
 ENVIRONMENT=development
-
+STRIPE_SECRET=sk_test_v9LWIQaVK4WQnNUmFzeTal7R
```

Comment:
Once we go live it should be simply stored in the env variable. For now, we can just use this one so that there is no need to remember to set env variable each time.